images/Win11-DevBox/build.pkr.hcl

```diff
@@ -23,7 +23,7 @@ source "azure-arm" "vm" {
   # base image options (Azure Marketplace Images only)
   image_publisher    = "microsoftwindowsdesktop"
   image_offer        = "windows-ent-cpc"
-  image_sku          = "win11-21h2-ent-cpc-m365"
+  image_sku          = "win11-23h2-ent-cpc-m365"
```

Comment:
May be we can variablize this as this changes frequently.

Reply:
Ive only forced an update due to the sys eng with west germany,  also Mustafa mentioned that he doesnt mind developers running an update over night on their devboxes in todays sprint planning

Reply:
If we're using variables anyway, this would be a good one to do. Image offer too. It'll give us the ability to jump forward and/or create new versions whenever we want without needing a PR each time.

Reply:
decided we wont update image